lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -44,6 +44,9 @@ class SynchronizeLdm < BaseAction
 
         description 'Specifies how to synchronize LDM and resolve possible conflicts'
         param :synchronize_ldm, instance_of(Type::SynchronizeLDM), required: false, default: 'diff_against_master_with_fallback'
+
+        description 'Allows to have include deprecated with model diff/view'
+        param :include_deprecated, instance_of(Type::BooleanType), required: false
```

Comment:
why don't we set default = false, similar to exclude_fact_rule

Reply:
Because the /model/view has always used value true for this call. So if set default = false. It will affect to this code.